File: sudo/src/main/java/com/roman/gurdan/sudo/util/LogUtil.java
package com.roman.gurdan.sudo.util;

import android.util.Log;

public class LogUtil {

    public static final String TAG = "SUDOKU";

    public static void v(String msg) {
        Log.v(TAG, msg);
    }

    public static void d(String msg) {
        Log.d(TAG, msg);
    }

    public static void e(String msg) {
        Log.e(TAG, msg);
    }

    public static void w(String msg) {
        Log.w(TAG, msg);
    }

}

File: base/src/main/java/com/roman/garden/sudo/base/util/LogUtil.kt
package com.roman.garden.sudo.base.util

import android.util.Log

class LogUtil {

    companion object {
        private val TAG: String = "SUDUKO"

        fun v(msg: String?) = Log.v(TAG, "$msg")

        fun d(msg: String?) = Log.d(TAG, "$msg")

        fun e(msg: String?) = Log.e(TAG, "$msg")

        fun w(msg: String?) = Log.w(TAG, "$msg")

    }

}